### PR TITLE
SWARM-1553 add parameter to skip all executions for multi-module projects

### PR DESCRIPTION
**Pull request by mzatko**
- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

**Comment by heiko-braun** [comment]
test this please

**Comment by heiko-braun** [comment]
Thanks for the contribution @mzatko !

**Comment by kenfinnigan** [comment]
This appears to have broken the Maven plugin tests we have.

Could you take a look at the cause?

**Comment by mzatko** [comment]
All 4 tests are passing for me, what is the failure or error?

**Comment by heiko-braun** [comment]
@mzatko  See here https://ci.wildfly-swarm.io/job/wildfly-swarm-pull-request-linux/94/

**Comment by mzatko** [comment]
@kenfinnigan , @heiko-braun  I am afraid I will need some details about the failure as all tests pass without any failure or error on my local environment

**Comment by mzatko** [comment]
@kenfinnigan , @heiko-braun this is probably a dumb question but what are prerequisites for running tests in wildfly-swarm/testsuite/testsuite-maven-plugin? if I run it with `mvn clean test` I only get skipped tests

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.wildfly.swarm.maven.plugin.MavenPluginTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 5, Time elapsed: 0.004 sec - in org.wildfly.swarm.maven.plugin.MavenPluginTest

Results :

Tests run: 5, Failures: 0, Errors: 0, Skipped: 5
```

**Comment by kenfinnigan** [comment]
If I recall it requires `MAVEN_HOME` to be set explicitly. Is that right @Ladicek ?

**Comment by Ladicek** [comment]
Yea the `M2_HOME` environment variable needs to be set.

**Comment by mzatko** [comment]
yeah, thank you @Ladicek, that was it. Now I can see failures, it looks like there is assertion for max. 2 WARNINGs about mainClass deprecation. It was somehow missing in `stop` goal so I added it there too and now it is logged 3 times during 2 of tests...
Do you think it is OK just to change in MavenPluginTest.java
`assertThat(count).as("There should only be 1 or 2 warnings").isIn(1, 2);`
to
`assertThat(count).as("There should only be 1 to 3 warnings").isIn(1, 2, 3);`
?
thank you

**Comment by kenfinnigan** [comment]
I think that should be fine

**Comment by Ladicek** [comment]
I don't think it's intentional for the `stop` goal to print the warning, but then again, I don't think it's a big deal. So if we agree with that, then the proposed change to the test is the right thing to do.

**Comment by mzatko** [comment]
So, I fixed it as proposed. Now even those additional tests are passing...

**Comment by kenfinnigan** [comment]
Awesome @mzatko, when you've pushed the changes and CI is green we will merge

Thanks again!

File: testsuite/testsuite-maven-plugin/src/test/java/org/wildfly/swarm/maven/plugin/MavenPluginTest.java [hunk 1]
```diff
@@ -267,7 +267,7 @@ public void doBuildUberjarAndRunTests() throws IOException, VerificationExceptio
             assertThat(log).contains("Custom main() usage is intended to be deprecated in a future release");
             // 1st warning for wildfly-swarm:package
             // 2nd warning possibly for wildfly-swarm:start for tests
-            assertThat(count).as("There should only be 1 or 2 warnings").isIn(1, 2);
+            assertThat(count).as("There should only be 1 to 3 warnings").isIn(1, 2, 3);
```
Comment (Ladicek) [inline testsuite/testsuite-maven-plugin/src/test/java/org/wildfly/swarm/maven/plugin/MavenPluginTest.java]: Could you please add a comment that the 3rd warning might come from `wildfly-swarm:stop`? Thanks!

**Comment by Ladicek** [comment]
Thanks! I wonder if I have the Jenkins powers...

**Comment by Ladicek** [comment]
test this please

File: testsuite/testsuite-maven-plugin/src/test/java/org/wildfly/swarm/maven/plugin/MavenPluginTest.java (hunk 1 above)
Comment (mzatko) [reply testsuite/testsuite-maven-plugin/src/test/java/org/wildfly/swarm/maven/plugin/MavenPluginTest.java]: I did :)

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**